indico/modules/events/editing/client/js/components/Section.jsx

```diff
@@ -0,0 +1,29 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Section({icon, label, description, children}) {
+  return (
+    <div className="section">
+      <span className={`icon icon-${icon}`} style={{alignSelf: 'center'}} />
```

Comment:
I don't think this style is needed anymore after the latest commit to center all icons

Reply:
indeed, removed it